### ISSUE TITLE
Number of read articles and statistics is wrong

### ISSUE THREAD

**Issue by ddeimeke** [wallabag/wallabag#2716]
:warning: If your issue is about an error during fetching a link, please read: http://doc.wallabag.org/en/master/user/errors_during_fetching.html#how-can-i-help-to-fix-that

### Issue details

It seems the read articles counter counts only the number of articles archived. That's wrong in my opinion since there is no need to archive an article after you read it.

I really read a lot of articles and I star them and mark them read if they go to my weekly Linkdump I publish in my blog. After publishing the articles I delete them in Wallabag since I do not need a separate archive of already read articles.

### Environment

* wallabag version (or git revision) that exhibits the issue 2.1.4

**Comment by j0k3r** [comment]
> It seems the read articles counter counts only the number of articles archived

This is the way it was intended to work.

> there is no need to archive an article after you read it

This is not true to every one. I mark an article as read (I mean archive it) as soon as I read it.

The "unread" section is a list of articles that user decide to _mark as read_. It's not a list of article you have actually read, and by read I mean you have scrolled until the end of the page so the system should know that you've read that article.

This is not the way it works in wallabag neither in the whole _read it later_ ecosystem  (as far as I know).

BTW it can be an improvment which might be enable as an option..

**Comment by ddeimeke** [comment]
I completely agree with you and don't think that all of us Wallabag users work the same way.

Honestly, I don't know how "reat it later" worked and it is not my "benchmark".

You explained exactly how I would expect it to work, "... scrolled until the end of the page so the system should know that you've read that article.", which should be implemented in the apps as well.

**Comment by j0k3r** [comment]
What if a button popup when you just arrive at the end of the article asking you to mark the article as read?
Because, sometimes, I'm just going to the end of an article just to check it's ok. Which doesn't mean, I've read it.

**Comment by ddeimeke** [comment]
Would be good as well.

**Comment by koichirose** [comment]
About this: I just installed wallabag and imported articles from Pocket.
This is what happens:
_Since 3 July 2017 you read 1269 articles. That is about 1269 a day!_ 

But it seems it would be impossible to fix this, by reading  Pocket's API docs there's no way to get the "archived_at" timestamp. (https://getpocket.com/developer/docs/v3/retrieve)

**Comment by Crocmagnon** [comment]
> About this: I just installed wallabag and imported articles from Pocket.
This is what happens:
Since 3 July 2017 you read 1269 articles. That is about 1269 a day!

I'm also affected by this issue, it renders the statistics unusable 😕 Also I just noticed the date is not in the same language as the test around 😅 

![image](https://user-images.githubusercontent.com/3630554/108812437-68a06200-75af-11eb-9dba-39954e7762a8.png)
